libfreerdp/core/errbase.c

```diff
@@ -0,0 +1,88 @@
+/**
+ * FreeRDP: A Remote Desktop Protocol Implementation
+ * Error Base
+ *
+ * Copyright 2015 Armin Novak <armin.novak@thincast.com>
+ * Copyright 2015 Thincast Technologies GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <stdio.h>
+
+#include <freerdp/log.h>
+
+#include "errinfo.h"
+
+#define TAG FREERDP_TAG("core")
+
+#define ERRBASE_DEFINE(_code)	    { ERRBASE_##_code , "ERRBASE_" #_code , ERRBASE_##_code##_STRING }
+
+/* Protocol-independent codes */
+
+#define ERRBASE_POSTCONNECTERROR_STRING \
```

Comment:
This looks like a leftover? (this might be problematic for some preprocessors because of the \ at the end)